docs/tutorials/my-first-custom-action.rst

```diff
@@ -0,0 +1,256 @@
+Your first custom action!
+######################################################
+
+Learn how to automate everything with Robusta, in *15 minutes* or less :)
```

Comment:
Perhaps be more specific than "everything"

e.g: "Learn how to automate _Kubernetes error handling_ with Robusta..." or some other detailed description